### ISSUE TITLE
Collisions order

### ISSUE THREAD

**Issue by Cqoicebordel** [arcusmaximus/YTSubConverter#60]
I'm trying to upload twitch videos to YT, with the chat as a subtitle.  
I have the json from Twitch which I used to generate a .ssa. Then I converted it manually to an .ass to test this converter.  
It worked quite well, and I was impressed to see that the collisions were combined.  
Except that, for my usage, the collisions were in reverse order. I'll explain :   
The subtitles appeared and disappeared in that order : 
```
First
---------
Second
First
---------
Third
Second
First
---------
Third
Second
---------
```

That is fine for most thing, but I'm trying to have a Twitch chat here, so for my use, the order should be
```
First
---------
First
Second
---------
First
Second
Third
---------
Second
Third
---------
```

I tried to change the setting in the header of the .ass, to have `Collisions` set to `Reverse` (vs the original `Normal`) but it didn't work (in fact, it looks like it isn't taken into account in any software, but that's another issue).  

So, my questions are : is what I'm looking for already possible ? Or is it a feature request you may consider ?  
Thanks !

**Comment by arcusmaximus** [comment]
The converter doesn't support reverse collisions right now because, well, I wasn't even aware this setting existed. Like you said, it does seem to be rather obscure - Aegisub's Properties window doesn't have it anymore, and if I manually add it to the .ass file, nothing changes in Aegisub or VLC.

That being said, it shouldn't be too much work to add support for this.

**Comment by arcusmaximus** [comment]
Support for this attribute is now available in the new 1.6.2 release.

**Comment by Cqoicebordel** [comment]
Thanks a lot !!!
